Products/ZenModel/FileSystem.py

```diff
@@ -110,11 +110,23 @@ class FileSystem(OSComponent):
         )
 
 
+    def getNoneRootTotalBlocks(self):
```

Comment:
If kept as a separate method, it needs to be renamed and _not_ be public because it's an implementation detail of the `getTotalBlocks` method.  I would suggest `_getSumOfPartsTotalBlocks` as a name that better describes what the value returns (with a `_` prefix to make it not public).